### PR TITLE
docs: update weekly-report-0524

### PR DESCRIPTION
**Pull request by Qizeng-api7**

### CODE REVIEW

File: blog/en/blog/2023/05/24/weekly-report-0524.md [hunk 1]
```diff
@@ -31,12 +31,12 @@ We have also sorted out some issues for newcomers to the community to participat
 
 ### Apache APISIX
 
-- [Proxy-mirror plugin provides support for grpc and grpcs traffic proxying](https://github.com/apache/apisix/pull/9388)（贡献者：[Sn0rt](https://github.com/Sn0rt)）
+- [Proxy-mirror plugin provides support for grpc and grpcs traffic proxying](https://github.com/apache/apisix/pull/9388)（contributor：[Sn0rt](https://github.com/Sn0rt)）
 
-- [Reuse etcd connections to improve etcd read-write throughput](https://github.com/apache/apisix/pull/9420)（贡献者：[kingluo](https://github.com/kingluo)）
+- [Reuse etcd connections to improve etcd read-write throughput](https://github.com/apache/apisix/pull/9420)（contributor：[kingluo](https://github.com/kingluo)）
```
Comment (Yilialinn) [inline blog/en/blog/2023/05/24/weekly-report-0524.md]:
```suggestion
- [Reuse etcd connections to improve etcd read-write throughput](https://github.com/apache/apisix/pull/9420)(contributor: [kingluo](https://github.com/kingluo))
```

File: blog/en/blog/2023/05/24/weekly-report-0524.md [hunk 2]
```diff
@@ -31,12 +31,12 @@ We have also sorted out some issues for newcomers to the community to participat
 
 ### Apache APISIX
 
-- [Proxy-mirror plugin provides support for grpc and grpcs traffic proxying](https://github.com/apache/apisix/pull/9388)（贡献者：[Sn0rt](https://github.com/Sn0rt)）
+- [Proxy-mirror plugin provides support for grpc and grpcs traffic proxying](https://github.com/apache/apisix/pull/9388)（contributor：[Sn0rt](https://github.com/Sn0rt)）
```
Comment (Yilialinn) [inline blog/en/blog/2023/05/24/weekly-report-0524.md]:
```suggestion
- [Proxy-mirror plugin provides support for grpc and grpcs traffic proxying](https://github.com/apache/apisix/pull/9388) (contributor: [Sn0rt](https://github.com/Sn0rt))
```

File: blog/en/blog/2023/05/24/weekly-report-0524.md [hunk 3]
```diff
@@ -31,12 +31,12 @@ We have also sorted out some issues for newcomers to the community to participat
...
 
-- [Reuse etcd connections to improve etcd read-write throughput](https://github.com/apache/apisix/pull/9420)（贡献者：[kingluo](https://github.com/kingluo)）
+- [Reuse etcd connections to improve etcd read-write throughput](https://github.com/apache/apisix/pull/9420)（contributor：[kingluo](https://github.com/kingluo)）
 
 ### Apache APISIX Ingress Controller
 
-- [Apache APISIX Ingress Controller project removes support for custom resources in version v2beta3](https://github.com/apache/apisix-ingress-controller/pull/1817)（贡献者：[tao12345666333](https://github.com/tao12345666333)）
+- [Apache APISIX Ingress Controller project removes support for custom resources in version v2beta3](https://github.com/apache/apisix-ingress-controller/pull/1817)（contributor：[tao12345666333](https://github.com/tao12345666333)）
```
Comment (Yilialinn) [inline blog/en/blog/2023/05/24/weekly-report-0524.md]:
```suggestion
- [Apache APISIX Ingress Controller project removes support for custom resources in version v2beta3](https://github.com/apache/apisix-ingress-controller/pull/1817) (contributor: [tao12345666333](https://github.com/tao12345666333))
```

**Review by Yilialinn — APPROVED** [review]
@guoqqqi @SkyeYoung Could you help to merge?

**Review by Gallardot — APPROVED** [review]

**Review by yzeng25 — APPROVED** [review]

**Comment by yzeng25** [comment]
Don't forget to add `docs: ` in the title next time, and you should add some descriptions to explain what you changed in this pull request. It really helps reviewers to better understand what you are trying to do in the pull request, especially when multiple files are involved.

**Comment by Yilialinn** [comment]
> Don't forget to add `docs: ` in the title next time, and you should add some descriptions to explain what you changed in this pull request. It really helps reviewers to better understand what you are trying to do in the pull request, especially when multiple files are involved.

cc @Qizeng-api7

**Pull request merged**